tools/run_tests/performance/templates/basic_template.yaml

```diff
@@ -0,0 +1,248 @@
+apiVersion: e2etest.grpc.io/v1
```

Comment:
yaml files support comments.
 Can the "loadtest_template.py" script generate a comment that has a note that the file was generated by "loadtest_template.py" and that it has client specs and server specs for multiple language and that the right spec will be selected based on the scenario's language when loadtest configurations are generated from this template?

Reply:
Done. I added a link to the README file as well, at the top of every template and to every configuration file.